src/Microsoft.Kusto.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -860,15 +858,17 @@ internal Hover GetHoverItem(TextDocumentPosition textDocumentPosition, ScriptFil
 
             ScriptDocumentInfo scriptDocumentInfo = new ScriptDocumentInfo(textDocumentPosition, scriptFile, scriptParseInfo);
 
-            if(connInfo != null){
+            if (connInfo != null)
```

Comment:
Fixed formatting and pulled SchemaState from the connection instead of dataSource